src/library/scala/collection/SeqView.scala

```diff
@@ -31,6 +31,9 @@ trait SeqView[+A] extends SeqOps[A, View, View[A]] with View[A] {
   def prependedAll[B >: A](prefix: SeqView.SomeSeqOps[B]): SeqView[B] = new SeqView.Concat(prefix, this)
 
   override def sorted[B >: A](implicit ord: Ordering[B]): SeqView[A] = new SeqView.Sorted(this, ord)
+
+  @deprecatedOverriding("Compatibility override", since="2.13.0")
+  override protected[this] def stringPrefix: String = "SeqView"
```

Comment:
Why not override `className` here like elsewhere?

Reply:
because someone in 2.12 might have overridden `stringPrefix`, as `SeqView` existed in 2.12